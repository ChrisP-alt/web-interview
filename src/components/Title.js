import React from 'react'

import styles from '../styles/components/title.module.scss'

const Title = () => (
  <div className={styles.wrapper}>
    <h1>New Appointment</h1>
  </div>
)

export default Title
