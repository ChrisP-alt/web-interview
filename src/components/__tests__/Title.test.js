import React from 'react'
import { shallow } from 'enzyme'

import Title from '../Title'

describe('Title component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Title />)
    expect(wrapper).toMatchSnapshot()
  })
})
