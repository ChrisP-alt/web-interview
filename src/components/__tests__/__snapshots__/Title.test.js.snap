// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component renders correctly 1`] = `
<div
  className="wrapper"
>
  <h1>
    New Appointment
  </h1>
</div>
`;
